js/index.js

```diff
@@ -40,3 +40,50 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Navigation loop through JSON
+const navigation = document.querySelectorAll('a');
+{
+  let i = 1;
+  navigation.forEach(navigation => {
+    navigation.textContent = siteContent.nav[`nav-item-${i}`];
+    navigation.style.color = "green";
+    i++;
+  });
+}
+
+// add two navigation things
```

Comment:
Looks like you started to label your sections and then stopped. Make sure you make a habit of that because it will be easier to go back and debug or change code and if someone else needs to work on your code they know what's going on and where.